losses/multiboxloss.py

```diff
@@ -0,0 +1,183 @@
+###################################################################################################
+#
+# Copyright (C) 2022 Maxim Integrated Products, Inc. All Rights Reserved.
+#
+# Maxim Integrated Products, Inc. Default Copyright Notice:
+# https://www.maximintegrated.com/en/aboutus/legal/copyrights.html
+#
+###################################################################################################
+"""
+    Multi-box Loss for Object Detection Models
+"""
+import torch
+from torch import nn
+
+import utils.object_detection_utils as obj_det_utils
+
+
+class MultiBoxLoss(nn.Module):
+    """
+        Multi-box Loss for Object Detection Models
+
+        Multi-box loss is a weighted sum of the loss components: location loss and confidence loss.
+
+        Location loss  measures the distance between the annotated object boxes and the predicted
+        bounding boxes. SSD paper uses smooth L1 loss for this purpose and so as the below
+        implementation.
+
+        Confidence Loss measures the perfroamnce of the  object classification per box.Categorical
+        cross-entropy is used to compute this loss.
+
+        SSD paper (Liu, W. et al. (2016). SSD: Single Shot MultiBox Detector,
+        https://doi.org/10.1007/978-3-319-46448-0_2)
+
+        Multi Nox Loss code is from GitHub repo:
```

Comment:
Please move the copyright into the file header as well just like you did for the other file

Reply:
Sure done and will be commit amended